lab4/provided/record_store.hh

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
Use the header guards like the other files. See https://google.github.io/styleguide/cppguide.html#The__define_Guard. `#pragma once` is not standard, though supported by most compilers.
Also https://stackoverflow.com/questions/1143936/pragma-once-vs-include-guards